svn/common.py

```diff
@@ -23,9 +23,9 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
 
         self.__type = type_
 
-    def run_command(self, subcommand, args, success_code=0, 
+    def run_command(self, subcommand, args, success_code=0,
                     return_stderr=False, combine=False, return_binary=False):
-# TODO(dustin): return_stderr is no longer implemented.
+        # TODO(dustin): return_stderr is no longer implemented.
```

Comment:
I am undoing this but any specific reason for undoing.. It wasn't looking good and was unindented before.